tests/README.md

```diff
@@ -41,24 +45,43 @@ Use [helm.sh](/tests/scripts/helm.sh) to install Helm and set up Rook charts def
 - To clean up `tests/scripts/helm.sh clean`.
 
 **NOTE:** These helper scripts depend on some artifacts under the `_output/` directory generated during build time.
-These scripts should be run from the project root. e.g., `tests/script/minikube.sh up`.
+These scripts should be run from the project root.
 
 **NOTE**: If Helm is not available in your `PATH`, Helm will be downloaded to a temporary directory (`/tmp/rook-tests-scripts-helm`) and used from that directory.
 
 ## Run Tests
 
 From the project root do the following after bringing up your Kubernetes cluster such as minikube:
 
-### 1. Build rook
+### 1. Start Kubernetes
 
-`make build`
+Start the minikube cluster and copy the local build into minikube:
 
-### 2. Start Kubernetes
+```console
+# On Linux
+minikube start --disk-size=40g --extra-disks=1 --driver kvm2
 
-Start the minikube cluster and copy the local build into minikube:
+# On MacOS
+minikube start --disk-size=40g --extra-disks=1 --driver hyperkit
+
+eval $(minikube docker-env -p minikube)
+```
+
+### 2. Build rook
+
+Since we have populated the Docker registry environment variables earlier, the build
```

Comment:
Maybe it is me, but i do not understand this point well. How we did that? can you explain better or add a link to the documentation where is explained how to populate these variables?

Reply:
Please look at the last version, the envs are populated with https://github.com/rook/rook/pull/9056/files#diff-dacac2ebf9792f0d23c0f922a744486ded01901957d5281290925acd89cf83acR59